mxcube3/ui/components/Tasks/DataCollection.js

```diff
@@ -87,11 +87,6 @@ class DataCollection extends React.Component {
            >
              Default Parameters
            </Button>
-           <Button bsSize="xsmall" bsStyle="default"
```

Comment:
why? Is it confusing to have both?

Reply:
Yes, the scientist here found it a bit confusing and questioned the need for doing reset and it was difficult to argue for keeping it.